awx/main/tasks.py

```diff
@@ -782,8 +786,9 @@ def add_ansible_venv(self, venv_path, env, add_awx_lib=True, **kwargs):
                 'a valid Python virtualenv does not exist at {}'.format(venv_path)
             )
         env.pop('PYTHONPATH', None)  # default to none if no python_ver matches
-        if os.path.isdir(os.path.join(venv_libdir, "python2.7")):
-            env['PYTHONPATH'] = os.path.join(venv_libdir, "python2.7", "site-packages") + ":"
+        for supported in ('python2.7', 'python3.6'):
```

Comment:
Do you feel like "AWX only supports ansible when run in Python 2.7 or Python 3.6" is a reasonable limitation?

Reply:
I may just make this smart enough to detect multiple ansible versions; otherwise somebody will want to run a py37 they build, and won't be able to.